### ISSUE TITLE
edge: reloading page at some point

### ISSUE THREAD

**Issue by beqabeqa473** [nvaccess/nvda#6223]
hello.

unfortunately, cannot test with other screenreaders for now, but in some webpages, going with quicknav by headings, page is reloaded in edge.

a good example of that is page: http://tiflocomp.ru/index.php?mod=news&id=1293&act=show

@michaelDCurran

**Comment by michaelDCurran** [comment]
I can no longer reproduce this in Edge in Windows build 15058.
